.circleci/config.yml

```diff
@@ -128,6 +128,24 @@ jobs:
           name: Test statics location in production image
           command: ./docker/tests/statics.sh marsha ${CIRCLE_SHA1}
 
+  build-lambda-docker:
+    machine:
+      image: ubuntu-2004:202010-01
+    working_directory: ~/marsha
+    steps:
+      # Checkout repository sources
+      - checkout
+      - *docker_login
+      # Each image is tagged with the current git commit sha1 to avoid
+      # collisions in parallel builds.
+      - run: 
+          name: Build production image
+          command: ./bin/lambda/build.sh ${CIRCLE_SHA1}
+
+      - run:
+          name: Check built image availability
+          command: docker image ls "${LAMBDA_IMAGE_NAME}:${CIRCLE_SHA1}"
```

Comment:
Didn't know about `docker image ls`, always used `docker images`:

```suggestion
          command: docker images "${LAMBDA_IMAGE_NAME}:${CIRCLE_SHA1}"
```

TIL: it's an alias.

Reply:
I prefer `docker image ls`, I know what I'm doint when I'm reading.

Reply:
No problem with that. :+1: